edenia/eden-spend-explorer
# Feat/create a help section 292

### Create a help section

### What does this PR do?

- Resolve #292 
- Resolve #325 

### Steps to test

1. Run Project
2. Enter to Help
3. Must see this:
![imagen](https://user-images.githubusercontent.com/49013556/208755741-60a335c3-449d-46b4-a038-2f6daaa0b11b.png)
![imagen](https://user-images.githubusercontent.com/49013556/208755782-4b447dd4-ca09-4bc4-a4d0-fe6993bc892b.png)


#### CheckList
